### PR TITLE
Support rebuild index plan

### PR DESCRIPTION
**Pull request by darionyaphet**
Create Tag and Index, writing some data into it then check the index by lookup sentence:

index_create_lookup gv

Create Tag and writing some data into it. After that create index and rebuild it. Finally check the index by lookup sentence:

rebuild_index gv

### CODE REVIEW

**Review by critical27 — APPROVED** [review]
LGTM

File: src/nebula/NebulaAction.h [hunk 1]
```diff
@@ -188,9 +188,44 @@ class WalkThroughAction : public core::Action {
     std::string  tag_;
     std::string  col_;
     uint64_t     totalRows_;
+    uint32_t     try_;
+    uint32_t     retryIntervalMs_;
     uint64_t     start_ = 0;
+};
+
+class LookUpAction : public core::Action {
```
Comment (dangleptr) [inline src/nebula/NebulaAction.h:196]: Maybe LookUpAction could be derived from MetaAction?

Comment (darionyaphet) [reply src/nebula/NebulaAction.h:196]: Actually `LookUpAction` extend `core::Action` and `CreateIndexAction` and `RebuildIndexAction` should extend `MetaAction`

**Review by critical27 — APPROVED** [review]
LGTM

**Pull request merged**